tests/modules/individuals/resources/test_elasticsearch.py

```diff
@@ -35,19 +36,21 @@ def test_individual_elasticsearch_mappings(
     EXPECTED_KEYS = {
         'created',
         'guid',
-        'birth',
         'customFields',
         'encounters',
         'has_annotations',
         'updated',
-        # 'social_groups',
         'indexed',
-        # 'names',
         'last_seen',
         'last_seen_specificity',
-        'death',
-        'comments',
+        'firstName_keyword',
+        'num_encounters',
         '_schema',
+        # 'comments',
```

Comment:
Is there a plan to recover or otherwise store `comments`, `names`, `social_groups`, `birth`, or `death`?

Reply:
Thanks for highlighthing this, that certainly needs fixing. Will talk to @naknomum  as this is still a problem and it's not clear to me why these fields are missing.